File: Leetcode_Java/0024.Swap Nodes in Two Pairs/src/ListNode.java

public class ListNode {
	int val;
	ListNode next;
	ListNode(int x){
		val = x;
	}
	
	/**
	 * Print all nodes, just for test
	 */
	@Override
	public String toString() {
	    String result = val + " ";
	    if (next != null) {
	        result += next.toString();
	    }
	    return result;
	}
}

File: Leetcode_Python/leet_code_problem_19_remove_nth_node_from_end_of_list/ListNode.py
class ListNode(object):
    def __init__(self, x):
        self.val = x
        self.next = None

    # This function prints all nodes, just for test
    def print_all_nodes(self):
        list_node = self  # start from the head node

        while list_node != None:
            print list_node.val
            list_node = list_node.next
